mlflow/tracking/client.py

```diff
@@ -2736,17 +2747,32 @@ def print_model_version_info(mv):
             --
             Name: RandomForestRegression
             Version: 1
-            Tags: {'t': '1'}
+            Tags: {'t': '1', 't1': '1'}
         """
+        _validate_model_version_or_stage_exists(version, stage)
+        if stage:
```

Comment:
If stage is specified, let's check `version` argument is None (if no raise ValueError), because if both `stage` and `version` parameters specified then it is ambiguous.

also document this point.

Reply:
@WeichenXu123  First part of your comment is already [implemented](https://github.com/mlflow/mlflow/pull/6185/files#diff-ee45120ef991391de7d556641c2dfd4a8841272bcd1e0f21a82c9081fbf8d746R392-R395). 
Second part of the comment, I will add details to doc string.